float64[] m    # Measurement vector
string sensor  # Name defining the sensor
string target  # Name defining the target (the thing being sensed)